Documentation/commands.md

```diff
@@ -72,7 +72,7 @@ In addition to the flags used by individual `rkt` commands, `rkt` has a set of g
 | --- | --- | --- | --- |
 | `--debug` |  `false` | `true` or `false` | Prints out more debug information to `stderr` |
 | `--dir` | `/var/lib/rkt` | A directory path | Path to the `rkt` data directory |
-| `--insecure-options` |  none | **none**: All security features are enabled<br/>**http**: Allow HTTP connections. Be warned that this will send any credentials as clear text.<br/>**image**: Disables verifying image signatures<br/>**tls**: Accept any certificate from the server and any host name in that certificate<br/>**ondisk**: Disables verifying the integrity of the on-disk, rendered image before running. This significantly speeds up start time.<br/>**all**: Disables all security checks | Comma-separated list of security features to disable |
+| `--insecure-options` |  none | **none**: All security features are enabled<br/>**http**: Allow HTTP connections. Be warned that this will send any credentials as clear text.<br/>**image**: Disables verifying image signatures<br/>**tls**: Accept any certificate from the server and any host name in that certificate<br/>**ondisk**: Disables verifying the integrity of the on-disk, rendered image before running. This significantly speeds up start time.<br/>**pubkey**: Allow fetching pubkeys via insecure connections (via HTTP connections or from servers with unverified certificates). The keys will be fetched to a temporary keystore regardless of whether the connection was secure or not.<br/>**all**: Disables all security checks | Comma-separated list of security features to disable |
```

Comment:
Yeah, at first I thought we should keep the potentially insecure keys outside of the main keystore, but following this logic, we should have been downloading potentially insecure images to the temporary store too. But since we are putting them into the main store, there is little sense in having a separate keystore.